frontend/src/utils/SpeakingDetector/speakingDetector.ts

```diff
@@ -89,7 +93,6 @@ class SpeakingDetector extends EventEmitter {
 
       this.source.connect(this.analyser);
 
-      this.analyser.fftSize = 2048;
```

Comment:
🤔 how come?

Reply:
this actually needs to be put back. I think I accidentally deleted it. apologies